### PR TITLE
translate 3 files to ko, update 2 files

### PR DESCRIPTION
**Pull request by ss-won**
en :
- [TypeScript with Web.js](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/playground-examples/copy/en/JavaScript/External%20APIs/TypeScript%20with%20Web.js)
- [Generic Functions.ts](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/playground-examples/copy/en/JavaScript/Functions%20with%20JavaScript/Generic%20Functions.ts)
- [Iterators and Generators.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/documentation/copy/en/reference/Iterators%20and%20Generators.md)

번역 리뷰 요청합니다 : @yeonjuan @guyeol @dvlprsh

ref #910

> 추가적으로 TypeScript with React.tsx, TypeScript with Node.js 부분도 수정했는데, 번역하다가 보니까 example: file-name 형태로 표기하지 않으면 playground 환경에서 링크가 생성이 안되는 것을 발견해서 기존 번역부분 수정했습니다.

image

### CODE REVIEW

File: packages/documentation/copy/ko/reference/Iterators and Generators.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: Iterators and Generators
+layout: docs
+permalink: /ko/docs/handbook/iterators-and-generators.html
+oneline: How Iterators and Generators work in TypeScript
+translatable: true
+---
+
+## 이터러블 (Iterables)
```
Comment (bumkeyy) [inline packages/documentation/copy/ko/reference/Iterators and Generators.md]:
```suggestion
## 이터러블
```

File: packages/documentation/copy/ko/reference/Iterators and Generators.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+[`Symbol.iterator`](Symbols.html#symboliterator) 프로퍼티에 대한 구현을 가진 객체는 이터러블로 간주합니다.
+`Array`, `Map`, `Set`, `String`, `Int32Array`, `Uint32Array`, 등과 같은 내장 타입에는 `Symbol.iterator` 프로퍼티가 이미 구현되어 있습니다.
+객체에 있는 `Symbol.iterator` 함수는 반복할 값 목록을 반환합니다.
+
+## `for..of`문
```
Comment (bumkeyy) [inline packages/documentation/copy/ko/reference/Iterators and Generators.md]:
```suggestion
## `for..of` 문
```

**Comment by bumkeyy** [comment]
LGTM

**Comment by orta** [comment]
That, err, seems wrong to me

**Comment by orta** [comment]
Fixed in https://github.com/OSS-Docs-Tools/code-owner-self-merge/commit/6dab34d48c14e2f29ec4b3b4ef56660f4528522f

**Pull request merged**